compio-quic/examples/http3-client.rs

```diff
@@ -0,0 +1,84 @@
+use std::{
+    net::{IpAddr, Ipv4Addr, Ipv6Addr, SocketAddr},
+    path::PathBuf,
+    str::FromStr,
+};
+
+use bytes::Buf;
+use compio_io::AsyncWriteAtExt;
+use compio_net::ToSocketAddrsAsync;
+use compio_quic::ClientBuilder;
+use http::{Request, Uri};
+use tracing_subscriber::EnvFilter;
+
+#[compio_macros::main]
+async fn main() {
+    tracing_subscriber::fmt()
+        .with_env_filter(EnvFilter::from_default_env())
+        .init();
+
+    let args = std::env::args().collect::<Vec<_>>();
+    if args.len() != 3 {
+        eprintln!("Usage: {} <URI> <OUT>", args[0]);
+        std::process::exit(1);
+    }
+
+    let uri = Uri::from_str(&args[1]).unwrap();
+    let outpath = PathBuf::from(&args[2]);
```

Comment:
How about output to stdout?

Reply:
I don't recommend it. Common QUIC testing sites contain very long html codes, specifically https://cloudflare-quic.com/, and they may contain indicators whether this page is accessed using HTTP/3.